adafruit_circuitplayground/express.py

```diff
@@ -671,17 +673,21 @@ def play_file(self, file_name):
         """
         # Play a specified file.
         self._speaker_enable.value = True
-        if sys.implementation.version[0] >= 3:
-            audio = audioio.AudioOut(board.SPEAKER)
-            file = audioio.WaveFile(open(file_name, "rb"))
-            audio.play(file)
-            while audio.playing:
-                pass
-        else:
-            audio = audioio.AudioOut(board.SPEAKER, open(file_name, "rb"))
-            audio.play()
-            while audio.playing:
-                pass
+        try:
+            if sys.implementation.version[0] >= 3:
+                audio = audioio.AudioOut(board.SPEAKER)
+                file = audioio.WaveFile(open(file_name, "rb"))
+                audio.play(file)
+                while audio.playing:
+                    pass
+                audio.deinit()
+            else:
+                audio = audioio.AudioOut(board.SPEAKER, open(file_name, "rb"))
+                audio.play()
+                while audio.playing:
+                    pass
+        except RuntimeError:
```

Comment:
This shouldn't require the try: except:. I think you need a `stop_tone()` call at the start of this in case one does:

```python
cpx.start_tone(440)
cpx.play_file("sample.wav")
```

I'd also suggest using `with` to manage making sure deinit is called:

```python
            if sys.implementation.version[0] >= 3:
                with audioio.AudioOut(board.SPEAKER) as audio:
                    wavefile = audioio.WaveFile(open(file_name, "rb"))
                    audio.play(wavefile)
                    while audio.playing:
                        pass
            else:
                with audioio.AudioOut(board.SPEAKER, open(file_name, "rb")) as audio:
                    audio.play()
                    while audio.playing:
                        pass
```

I also renamed file to wavefile because file is/was a python keyword which can make things weird later.